man/systemd.netdev.xml

```diff
@@ -1457,6 +1457,15 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>TlbDynamicLb=</varname></term>
```

Comment:
urks, what does "Tlb" and "Lb" stand for? I am left very confused about the naming of this option

Reply:
:)  LB-> Load Balancing
    T->Transmit

Reply:
yes nice.